ackermann_steering_controller/include/ackermann_steering_controller/odometry.h

```diff
@@ -0,0 +1,210 @@
+/*********************************************************************
```

Comment:
How does this differ from https://github.com/ros-controls/ros_controllers/blob/melodic-devel/diff_drive_controller/include/diff_drive_controller/odometry.h?
Please use another include guard

Reply:
Fixed at https://github.com/ros-controls/ros_controllers/pull/356/commits/be7d149661b978313a123a46c4539b80f9d4909f

ackermann's odometry differs from diff_drive's as follows

- parameters
  - It needs angle variable for steering joint.
- numerical formula of integral
  - It uses completely different computation because using angle parameter of steering joint.

Reply:
These differences are a good starting point for refactoring :)